ansible_collections/arista/avd/docs/stylesheets/tables.js

```diff
@@ -2,6 +2,5 @@
 document$.subscribe(function() {
   var tables = document.querySelectorAll("article table:not([class])")
   tables.forEach(function(table) {
-    new Tablesort(table)
   })
```

Comment:
I am not sure this is useless. If `Tablesort` updates the `table` item and attaches to it.

Reply:
Indeed it seems to be how it works: https://tristen.ca/tablesort/demo/

Reply:
So please find out how to exclude this from sonarlint. Something similar to # NOSONAR for Python.